ide/coqide_main.ml4

```diff
@@ -68,6 +69,7 @@ let set_win32_path () =
    is never read (by default) or to a temp log file (when in debug mode).
 *)
 
+[@@@ocaml.warning "-32"]
```

Comment:
What's going on here?

Reply:
Conditional compilation (used only on Win32).

Reply:
>Conditional compilation (used only on Win32).

OK I see it.